shared-components/tl-tournament-card.js

```diff
@@ -133,12 +93,42 @@ export class TournamentCard extends LitElement {
 
                 .button-label-large {
                     --mdc-typography-button-font-family: var(--font-family);
-                    --mdc-typography-button-font-size: var(--button-large-label-font-size);
-                    --mdc-typography-button-line-heightt: var(--button-large-label-line-height);
-                    --mdc-typography-button-font-weight: var(--button-large-label-font-weight);
+                    --mdc-typography-button-font-size: var(--button-large-font-size);
+                    --mdc-typography-button-line-height: var(--button-large-line-height);
+                    --mdc-typography-button-font-weight: var(--button-large-font-weight);
                     --mdc-typography-button-text-transform: none;
                     --mdc-typography-button-letter-spacing: none;
                 }
+
+                .event-title,
+                .event-text {
+                    margin: 0 2rem;
+                }
+
+                .event-title {
+                    margin-top: 1.2rem;
+                }
+
+                .event-prize {
+                    margin: 2rem 2rem 0;
+                }
+
+                .event-dates-label {
+                }
+
+                .event-prize-amount {
+                    background-color: #c078e4;
+                    padding: 0.2rem 1.8rem;
+                    margin-left: 1rem;
+                    border-radius: 2rem;
+                }
+
+                .details-button {
+                    position: absolute;
+                    align-self: flex-end;
+                    bottom: 10%;
```

Comment:
If the positioning `absolute` is fine, the set of `bottom: 10%` does not work because the button is in the vertically center on the right, when compared to the text on the left... A longer description is going to make the area taller and then the `10%` will be off-track...